src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -47,8 +47,6 @@ object Settings {
              urn: String,
              doi: String,
              otherAccessDoi: Boolean,
-             stageFileDataAsRedirectDatastreams: Boolean,
-             fileDataRedirectBaseUrl: Option[URL],
```

Comment:
404: Pending the merge of https://github.com/DANS-KNAW/easy-ingest-flow/pull/17 I created the temporary [PR#19](https://github.com/DANS-KNAW/easy-ingest-flow/pull/19/files#diff-6536be7e2937bc0d681776a488976215)